src/ledmon.c

```diff
@@ -263,6 +267,41 @@ static void _ledmon_help(void)
 }
 
 /**
+ * @brief Prints given controllers list.
+ *
+ * This is internal function of monitor service. This function prints the path
+ * and type of controller for every controller on given in argument list.
+ *
+ * @param[in]      cntrl_device           address to first element from
+ *                                        controllers list.
+ *
+ * @return The function does not return a value.
+ */
+static void _print_controllers(struct cntrl_device *ctrl_list)
+{
+	while (ctrl_list != NULL) {
+		printf("%s ", ctrl_list->sysfs_path);
+		switch (ctrl_list->cntrl_type) {
+		case CNTRL_TYPE_DELLSSD:
+			printf("(DELL SSD)\n");
+			break;
+		case CNTRL_TYPE_VMD:
+			printf("(VMD)\n");
+			break;
+		case CNTRL_TYPE_SCSI:
+			printf("(SCSI)\n");
+			break;
+		case CNTRL_TYPE_AHCI:
+			printf("(AHCI)\n");
+			break;
+		default:
+			printf("\n");
+		}
+		ctrl_list = list_next(ctrl_list);
+	}
```

Comment:
I would prefer to create some look-up table with those names, something similiar to ibpi_str[] in ledmon.c